src/Process/Pool/Logger/Message/Process/FromProcessInterface.php

```diff
@@ -0,0 +1,48 @@
+<?php
+declare(strict_types=1);
+
+namespace Neighborhoods\Kojo\Process\Pool\Logger\Message\Process;
+
+use Neighborhoods\Kojo\ProcessInterface;
+
+class FromProcessInterface implements FromProcessInterfaceInterface
```

Comment:
I think this should be `Process\Pool\Logger\Message\Process`, only the actual builder's class name will specify that it's "from a process"